packages/cms-lib/lib/config.js

```diff
@@ -308,7 +367,7 @@ const setDefaultConfigPathIfUnset = () => {
 };
 
 const setDefaultConfigPath = () => {
-  setConfigPath(`${getCwd()}/${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME}`);
+  setConfigPath(`${os.homedir()}/${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME}`);
```

Comment:
Default config dir is now user's home dir.

Reply:
I'm wondering if we should skip writing `.gitignore` if the config is in home dir.  Assuming that wouldn't be in source control.

Reply:
I don’t think we should write to gitignore and leave that to users.